packages/expo-application/ios/ApplicationModule.swift

```diff
@@ -0,0 +1,45 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+import ExpoModulesCore
+
+public class ApplicationModule: Module {
+  public func definition() -> ModuleDefinition {
+    Name("ExpoApplication")
+
+    Constants([
+      "applicationName": Bundle.main.object(forInfoDictionaryKey: "CFBundleDisplayName"),
+      "applicationId": Bundle.main.object(forInfoDictionaryKey: "CFBundleIdentifier"),
+      "nativeApplicationVersion": Bundle.main.infoDictionary?["CFBundleShortVersionString"],
+      "nativeBuildVersion": Bundle.main.infoDictionary?["CFBundleVersion"]
+    ])
```

Comment:
```suggestion
 Constants {
      let infoPlist = Bundle.main.infoDictionary
      return [
        "applicationName": infoPlist?["CFBundleDisplayName"],
        "applicationId": infoPlist?["CFBundleIdentifier"],
        "nativeApplicationVersion": infoPlist?["CFBundleShortVersionString"],
        "nativeBuildVersion": infoPlist?["CFBundleVersion"]
      ]
    }
```
Why are we using different access patterns?

Reply:
Because I migrated that exactly from the Objective-C code. I changed it.

Reply:
I wouldn't migrate 1:1, if you see obvious issues feel free to change